src/components/PersonLink.tsx

```diff
@@ -0,0 +1,56 @@
+import { Link, useLocation, useParams } from 'react-router-dom';
+import { Person } from '../types';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person;
+  findParentPerson: (parentName: string) => Person | undefined;
+};
+
+const getLinkClass = (person: Person) => {
+  return classNames({ 'has-text-danger': person.sex === 'f' });
+};
+
+export const PersonLink: React.FC<Props> = ({ person, findParentPerson }) => {
+  const { slug } = useParams();
+  const location = useLocation();
+
+  const renderParentLink = (parentName: string | null) => {
```

Comment:
Consider renaming the `renderParentLink` function to `getParentLink`. The word 'render' is usually used for component methods which return JSX, while this function could be used outside of the JSX context.